src/com/palmergames/bukkit/towny/object/Nation.java

```diff
@@ -87,9 +88,15 @@ public boolean hasMutualAlly(Nation nation) {
 		return getAllies().contains(nation) && nation.getAllies().contains(this);
 	}
 
-	public boolean IsAlliedWith(Nation nation) {
-
-		return getAllies().contains(nation);
+	/**
+	 * Check if the targetNation is an Ally.
+	 * @param targetNation Any other nation than this one.
+	 * @return Whether the targetNation is an ally.
+	 * @deprecated Unused. Marked deprecated as of 0.97.4.1+. Use {@link #isAlliedWith(Nation)}.
+	 */
+	@Deprecated
+	public boolean IsAlliedWith(Nation targetNation) {
+		return isAlliedWith(targetNation);
```

Comment:
this appears to be returning itself instead of a new method?

Reply:
Actually, no. There are *two* `isAlliedWith(Nation)` checks. The only difference was in the capitalization of the method.
As this method was unused by anything internal to Towny, I marked it as deprecated and changed its target to the more standardized / actually used method. The intent here is to remove it.

Reply:
Ah I don't know how I missed the `i` cause I looked at that for a while before making this comment.